frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -56,7 +56,7 @@ export const EmailFormSection = ({
         <FormLabel
           isRequired
           useMarkdownForDescription
-          description="Add at least **2 recipients** to prevent loss of response. Learn more [how to guard against bounce emails](https://go.gov.sg/form-prevent-bounce)."
+          description="Add at least **2 recipients** to prevent loss of response. Learn more on [how to guard against email bounces](https://go.gov.sg/form-prevent-bounce)."
```

Comment:
ah, missed URL here!! should we export the URL into a constant in https://github.com/opengovsg/FormSG/blob/form-v2/develop/frontend/src/constants/links.ts#L5 and use it here?

Reply:
ok!